myapp/app/controllers/tasks_controller.rb

```diff
@@ -5,25 +5,27 @@ class TasksController < ApplicationController
   MSG_UPDATE_SUCCESS = 'Updated a task successfully.'
   MSG_UPDATE_FAILURE = 'Failed to update a task.'
   MSG_DELETE_SUCCESS = 'Deleted a task successfully.'
-  MSG_DELETE_FAILURE = 'Failed to delete a task.' # not in used
+  MSG_DELETE_FAILURE = 'Failed to delete a task.'
 
   def index
     @new_task = Task.new
-    @tasks = Task.all
+    @tasks = Task.order('created_at DESC')
   end
 
   def create
-    logger.debug { "params => #{params}" }
     data = { title: params[:task][:title], description: params[:task][:description]
 }
     @task = Task.new(data)
     if @task.save
-      flash[:notice] =  MSG_CREATE_SUCCESS
+      flash[:success] = MSG_CREATE_SUCCESS
```

Comment:
As per Ruby on Rails style, we set the string or I18n directly in the line instead of using a defined variable.

Reply:
Thanks for the suggestion. Than means following is better way as RoR best practice?

```
      flash[:success] = I18n.t 'msg_create_success'
```
and define 'msg_create_success' is in config/locales/ja.yml

```
ja:
  title: "タイトル"
  description: "詳細"
  created_at: "作成日時"
  updated_at: "更新日時"
  msg_create_success: '作成に成功しました'
  msg_create_failure: '作成に失敗しました'
  msg_update_success: '更新に成功しました'
  msg_update_failure: '更新に失敗しました'
  msg_delete_success: '削除に成功しました'
  msg_delete_failure: '削除に失敗しました'
```

Reply:
Yes, correct!

Reply:
Ok thanks, I addressed [it](https://github.com/Fablic/training/pull/1706/commits/138f123b42d5e2ec8d627b7c3dd52c6eb71bacc4)